scripts/lib-franklin.js

```diff
@@ -196,22 +196,26 @@ export async function fetchPlaceholders(prefix = 'default') {
   const loaded = window.placeholders[`${prefix}-loaded`];
   if (!loaded) {
     window.placeholders[`${prefix}-loaded`] = new Promise((resolve, reject) => {
-      try {
-        fetch(`${prefix === 'default' ? '' : prefix}/placeholders.json`)
-          .then((resp) => resp.json())
-          .then((json) => {
-            const placeholders = {};
-            json.data.forEach((placeholder) => {
+      fetch(`${prefix === 'default' ? '' : prefix}/placeholders.json`)
+        .then((resp) => {
+          if (resp.ok) {
+            return resp.json();
+          }
+          throw new Error(`${resp.status}: ${resp.statusText}`);
+        }).then((json) => {
+          const placeholders = {};
+          json.data
+            .filter((placeholder) => placeholder.Key && placeholder.Text)
```

Comment:
Filtering for existing `Key` is certainly useful, but making `Text` "truthiness" mandatory might be breaking backward compatibility: I think placeholders may also be empty strings under certain circumstances.

Reply:
Thanks, that makes sense - I think it is not important to filter based on an existing `Text`. The filtering on existing `Key` is the important bit. @rofe, do you want me to remove the `&& placeholder.Text` from the filter?

Reply:
yes, please 🙏